specifications/api/swagger.yaml

```diff
@@ -5259,6 +5340,8 @@ paths:
       summary: Get
       description: |
         <p align="right">status: <strong>⚠ under development</strong></p>
+
+        **TODO:** ADP-866 document this endpoint.
```

Comment:
I put these TODO messages in for you ;-)

Reply:
done